content/installation/net/configuration/YAML-properties-net.md

```diff
@@ -0,0 +1,202 @@
+<!--
+title: "YAML configuration for the .NET agent "
+description: "Instructions and template for configuring .NET agent properties via YAML file"
+tags: "installation net agent YAML configuration rules properties"
+-->
+
+Contrast support YAML-based configuration for the .NET agent. This allows you to store configuration on disk with environment variable and CLI args. overrides.
+
+> **Note:** While property formatting in YAML configuration files is shared by all Contrast agents, each agent must use its specified file. 
+
+## Load Path
+
+Configuration values use the following order of precedence: 
+
+1. Corporate rule (e.g., expired license overrides `assess.enable`)
+2. Command line value
+3. Specific environmental variable
+4. Generic environment variable value
+5. User configuration file value
+6. Contrast UI value
+7. Default value 
+
+Go to the [.NET YAML Template](installation-netconfig.html#net-template) for fully formatted properties that you can copy and use in your own configuration files. 
+
+## Set as System Properties
+
+You can also set all of the following YAML properties as system properties. Derive the system property key from the YAML by joining every node with a "." until you reach the bottom property. 
+
+> **Example:** If you want to override the `contrast` property, as given below, you can set `-Dcontrast.enable=false` as a system property.
+  * contrast: 
+    * enable: `true`
+
+<!-- The precedence and system properties example are copied from Java. Edit or delete as needed.  -->
```

Comment:
@DanFiedler If the precedence and system properties info is good, I'll delete this comment.

Reply:
@kaitlynmortimer .NET doesn't support command line arguments. (which is fine because Windows/IIS users don't typically use command line arguments.)